todo_app/data/mongo_items.py

```diff
@@ -0,0 +1,28 @@
+from bson import ObjectId
+
+from todo_app.classes.item import Item
+from todo_app.classes.mongodb_service import MongoDbService
+
+def get_items():
```

Comment:
Just in case of interest, not important to change here!

You may notice that the app is fairly slow after the shift to Mongo; although DB calls over the network can be part of the issue, here it's actually more to do with the fact that every call to one of these methods sets up a fresh connection with the DB, rather than sharing connections. One way you can test this out, and check how it improves locally is by converting this file to a class (`ItemService`?) that sets up a single connection to be reused:
```python
class ItemService:
    def __init__(self):
        self.mongoDbService = MongoDbService()

    def get_items(self):
        mongo_documents = list(self.mongoDbService.todo_items.find())
        ...
```
and then update where it's used in `app.py`:
```python
def create_app():
    app = Flask(__name__)
    app.config.from_object(Config())
    item_service = ItemService()

    @app.route('/')
    def index():
        items_view_model = ViewModel(item_service.get_items())
```